setup.py

```diff
@@ -70,15 +76,16 @@ def read_file(*path):
     zip_safe=False,  # otherwise ZConfig can't see component.xml
     install_requires=[
         'perfmetrics',
-        'ZODB3>=3.7.0',
+        'ZODB3>=3.7.0,<3.11.0',
```

Comment:
True. But as you have experimented on .travis.yml

```
# setup-ENV likes to update ZODB. But RecoveryStorage can't import
# handle_serials in 5.0, and there are transaction changes that affect
# our tests in 4.3+ (checkUndoCreationBranch1). It's hard to write a requirement like this in
# setup.py because of the ZODB3 meta distribution being renamed back
# to ZODB.
  - pip uninstall -y ZODB
  - pip install 'ZODB<4.3'
```
could be a hell. 

Don't  you think that 1.6 could be only for ZODB3<3.11, and other people must to use Relstorage 2.*?